CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 **Bug Fixes**:
 
 - Fix a bug where unreal crash reports were dropped when metrics extraction is enabled. ([#1355](https://github.com/getsentry/relay/pull/1355))
+- Extract user from metrics correctly. ([#1363](https://github.com/getsentry/relay/pull/1363))
```

Comment:
nit: I was wondering how to make the PR description a bit more specific while keeping it short:
```suggestion
- Extract user from metrics with EventUser's priority. ([#1363](https://github.com/getsentry/relay/pull/1363))
```